api/src/main/java/ru/urururu/util/Coverage.java

```diff
@@ -27,9 +39,24 @@ public static synchronized void markAsCode(File file, int line) {
         coverageInfo.set(line, oldInfo == null ? 0 : oldInfo);
     }
 
-    public static synchronized void hit(File file, int line) {
+    private static synchronized void hit(File file, int line) {
         List<Integer> coverageInfo = coverage.get(file);
-        coverageInfo.set(line, coverageInfo.get(line) + 1);
+
+        if (coverageInfo.size() <= line) {
+            LOGGER.warn("Line: " + line + ", Size: " + coverageInfo.size());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ", Size: " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Line: " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Invoke method(s) only conditionally. Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2629)